src/init.cpp

```diff
@@ -1389,10 +1389,13 @@ bool AppInitMain(const util::Ref& context, NodeContext& node, interfaces::BlockA
     assert(!node.connman);
     node.connman = MakeUnique<CConnman>(GetRand(std::numeric_limits<uint64_t>::max()), GetRand(std::numeric_limits<uint64_t>::max()), args.GetBoolArg("-networkactive", true));
 
+    assert(!node.fee_estimator);
+    node.fee_estimator = MakeUnique<CBlockPolicyEstimator>();
```

Comment:
cc21167415aa01db862035cd07b6b9b35fb53f35

would be nice if new code used `std::make_unique` instead of the deprecated wrapper

Reply:
Included, had to rebase anyway. Used it for CTxMemPool too as it was free diff-wise.